src/components/application_manager/src/message_helper.cc

```diff
@@ -2485,38 +2533,45 @@ bool MessageHelper::PrintSmartObject(const smart_objects::SmartObject& object) {
            key++) {
         ++tab;
 
-        printf("\n%s%s: ", tab_buffer.c_str(), (*key).c_str());
+        LOGGER_INFO(logger_,
+                    "\n" << tab_buffer.c_str() << (*key).c_str() << ": ");
         if (!PrintSmartObject(object[(*key).c_str()])) {
-          printf("\n");
+          LOGGER_INFO(logger_, "\n");
           return false;
         }
       }
       break;
     }
     case NsSmartDeviceLink::NsSmartObjects::SmartType_Boolean:
-      object.asBool() ? printf("true\n") : printf("false\n");
+      if (object.asBool()) {
+        LOGGER_INFO(logger_, "true\n");
+      } else {
+        LOGGER_INFO(logger_, "false\n");
+      }
       break;
     case NsSmartDeviceLink::NsSmartObjects::SmartType_Double: {
-      printf("%f", object.asDouble());
+      LOGGER_INFO(logger_, object.asDouble());
       break;
     }
     case NsSmartDeviceLink::NsSmartObjects::SmartType_Integer:
       break;
     case NsSmartDeviceLink::NsSmartObjects::SmartType_String:
-      printf("%s", object.asString().c_str());
+      LOGGER_INFO(logger_, object.asString().c_str());
       break;
     case NsSmartDeviceLink::NsSmartObjects::SmartType_Character:
-      printf("%c", object.asChar());
+      LOGGER_INFO(logger_, object.asChar());
       break;
     default:
-      printf("PrintSmartObject - default case\n");
+      LOGGER_INFO(logger_, "PrintSmartObject - default case\n");
       break;
   }
 
   if (0 != tab) {
     --tab;
   } else {
-    printf("\n-------------------------------------------------------------\n");
+    LOGGER_INFO(
+        logger_,
```

Comment:
@AGaliuzov I made those changes as required from `TODO` comment made from @LuxoftAKutsan . Now I would like @LuxoftAKutsan to review the MessageHelper::SendHashUpdateNotification method.